api/src/main/java/org/pmiops/workbench/rawls/RawlsConfig.java

```diff
@@ -32,7 +32,11 @@ public class RawlsConfig {
   public static final String END_USER_LENIENT_TIMEOUT_API_CLIENT =
       "rawlsEndUserLenientTimeoutApiClient";
   public static final String SERVICE_ACCOUNT_API_CLIENT = "rawlsServiceAccountApiClient";
-  public static final String SERVICE_ACCOUNT_WORKSPACE_API = "rawlsWorkspaceAclsApi";
+  public static final String SERVICE_ACCOUNT_LENIENT_TIMEOUT_API_CLIENT =
+      "rawlsServiceAccountLenientTimeoutApiClient";
+  public static final String SERVICE_ACCOUNT_WORKSPACE_API = "rawlsServiceAccountWorkspacesApi";
```

Comment:
the **values** of these strings are arbitrary, so it's OK to change them as long as we consistently use the consts.